framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/settings/builder/SetttingsBuilder.java

```diff
@@ -13,8 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.kie.cloud.openshift.constants;
+package org.kie.cloud.api.settings.builder;
 
-public enum ScenarioConstants {
-    KIE_SERVER_TEMPLATE_KEY, SMART_ROUTER_TEMPLATE_KEY, WORKBENCH_TEMPLATE_KEY, DATABASE_TEMPLATE_KEY
+/**
+ * Cloud settings builder. Create settings for Deployment scenario.
+ *
+ * @param <T> Setup to be built e.g. DeploymentSettings
+ * @see org.kie.cloud.api.settings.DeploymentSettings
+ */
+public interface SetttingsBuilder<T> {
```

Comment:
Looks like this builder is only ever used to build DeploymentSettings. Do we need this generic type parameter?
Also lookst like you have typo Setttings (3x 't')